src/test/java/com/dougnoel/sentinel/elements/ElementTests.java

```diff
@@ -179,4 +181,41 @@ public void sendSpecialKey() {
 		assertTrue("Expecting key press input to be empty.", input.getText().isEmpty());
 	}
 	
+	@Test
+	public void checkParentColorsOfTransparentElement() {
+		JavascriptExecutor js = (JavascriptExecutor) WebDriverFactory.getWebDriver(); 
+		
+		Color startingColor = Color.white;
+		String hexOfStartingColor = "#"+Integer.toHexString(startingColor.getRGB()).substring(2);
+		Color endingColor = Color.blue;
+		String hexOfEndingColor = "#"+Integer.toHexString(endingColor.getRGB()).substring(2);
+		
+		BaseSteps.navigateToPage("TextboxPage");
+		js.executeScript("arguments[0].style.backgroundColor=arguments[1];", getElement("Body").element(), hexOfStartingColor);
+		Assert.assertEquals(startingColor, getElement("Car Checkbox").getBackgroundColor());
+		js.executeScript("arguments[0].style.backgroundColor=arguments[1];", getElement("Body").element(), hexOfEndingColor);
+		Assert.assertEquals(endingColor, getElement("Car Checkbox").getBackgroundColor());
```

Comment:
There are two assertions in this unit test. These should be two separate tests.

Reply:
Done